packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1595,6 +1676,26 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.EditableText.contentInsertionConfiguration}
+  /// Configuration of handler for rich content inserted via the system input
+  /// method.
+  ///
+  /// By default, no handler will be created for inserted content. Set
+  /// [ContentInsertionConfiguration.onContentInserted] to provide a handler.
+  /// Additionally, set [ContentInsertionConfiguration.contentInsertionMimeTypes]
+  /// to limit the allowable mime types for inserted content.
+  ///
+  /// Note that if [contentInsertionConfiguration] is not provided, by default
+  /// an empty list of mime types will be sent to the Flutter Engine.
+  /// A handler function must be provided in order to customize the allowable
+  /// mime types for inserted content.
+  ///
+  /// If rich content is inserted without a handler, the system will display
+  /// a message informing the user that the current text input does not support
+  /// inserting rich content.
+  /// {@endtemplate}
+  final ContentInsertionConfiguration? contentInsertionConfiguration;
```

Comment:
This is also super clear compared to before.  If you don't pass in contentInsertionConfiguration, you don't want to enable the feature, and if you do, you do want to enable it.  Stupid simple.